contracts/pallet-ibc/ping/src/lib.rs

```diff
@@ -182,6 +181,11 @@ impl<T: Config + Send + Sync> Module for IbcModule<T> {
 		version: &Version,
 		counterparty_version: &Version,
 	) -> Result<Version, Ics04Error> {
+		assert_eq!(
+			&PortId::from_str(PORT_ID).expect("PORT_ID is static and valid; qed"),
+			port_id,
+			"routing is correct; qed"
+		);
```

Comment:
we shouldn't panic in the substrate runtime

Reply:
also this assertion is incorrect

Reply:
I have replaced assert with error.

Reply:
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/757125/200576882-9a7223da-e7ef-49de-82d8-4865003a1e4d.png">
it was this error before. I made different message because `port_id` should be my static port_id? it is not the rule that counter party port will be same as my port id?

Reply:
Counterparty.port_id is is port to which I will send packets, but `port_id` as parameter to method is my port id. I do not see error in assertion text.

Reply:
🤔

Reply:
after reading more and oding - assertion is correct.

Reply:
it was incorrect, and i fixed it.